.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 3.2.2
```

Comment:
we dont keep track using .tool-versions so delete this ig or add this to gitignore file

Reply:
okay